### DISCUSSION TITLE
DMA configuration with Viper

### DISCUSSION THREAD

**Discussion by sk8board** [micropython/micropython#16146] [Using MicroPython]
I am using Viper to configure DMA registers on an RP2040. Is there a way to configure a DMA interrupt to a handler function using Viper?

**Comment by GitHubsSilverBullet** [comment]
Is there any particular reason why you don't use the built-in functionality? IRQs are provided therein.

**Comment by sk8board** [comment]
I agree the IRQ functions in MicroPython are robust. 

I am learning how to use Viper and I like the low level control of configuring a DMA channel with Viper. While going down this path, I realized I do not know how, or if possible, to use Viper to configure a DMA interrupt handler. 

Is there a way to configure a DMA interrupt to a handler function using Viper?
